### ISSUE TITLE
how to make large kernel work?

### ISSUE THREAD

**Issue by TheGreatTreatsby** [DingXiaoH/RepLKNet-pytorch#29]
i find the input image size is quiet huge when conduct downstream task,Is this necessary?If training on small pictures, such as 416*416, can the large convolution kernel model still maintain its advantage?
In addition, for the size of the data set, without pre-training on large data sets, only training in small data sets, is the advantage of the large convolution kernel model still?

**Comment by DingXiaoH** [comment]
1. I find the input image size is quiet huge when conduct downstream task,Is this necessary?
We don't know. You may ask the creators of the datasets.
2. If training on small pictures, such as 416*416, can the large convolution kernel model still maintain its advantage?
The answer is yes, as shown in our paper (Guideline 5).
3. without pre-training on large data sets, only training in small data sets, is the advantage of the large convolution kernel model still?
4. We don't know. We have not tried big models without pretraining.
